### ISSUE TITLE
[BUG]: White text on orange background not enough contrast

### ISSUE THREAD

**Issue by ryandotfurrer** [TheTechCommute/TheTechCommute#30]
**Describe the bug**
The white text in the email signup box does not have enough contrast as per WCAG a11y standars.

**To Reproduce**
1. Go to homepage
2. Scroll down to email signup box
3. See error

**Expected behavior**
Should have enough contrast between text and background element.

**Screenshots**
![Arc Tech Commute-000108](https://github.com/user-attachments/assets/2cb53d79-21d2-46b9-b25e-01981da54b2e)

**Comment by Anush0108** [comment]
@ryandotfurrer Hey, can you assign this issue to me?

**Comment by ryandotfurrer** [comment]
> @ryandotfurrer Hey, can you assign this issue to me?
> 

I cannot. Maybe @jasonetorres or @elliezub can?
